datahub-web-react/src/app/entity/chart/preview/ChartPreview.tsx

```diff
@@ -76,7 +79,7 @@ export const ChartPreview = ({
             name={name || ''}
             urn={urn}
             description={description || ''}
-            type="Chart"
+            type={capitalizeFirstLetterOnly(subType) || 'Chart'}
```

Comment:
does capitalizeFirstLetterOnly handle case of undefined | null/

im assuming yes

Reply:
yes 🖖 

https://github.com/datahub-project/datahub/blob/399e032dfa2b4bf87b7b406e7b009e34e99a1003/datahub-web-react/src/app/shared/textUtil.ts#L17